packages/synthetic-chain/src/cli/run.ts

```diff
@@ -1,14 +1,12 @@
 import { execSync } from 'node:child_process';
 import { ProposalInfo, imageNameForProposal } from './proposals.js';
 
-export const runTestImages = (proposals: ProposalInfo[]) => {
-  for (const proposal of proposals) {
-    console.log(`Running test image for proposal ${proposal.proposalName}`);
-    const { name } = imageNameForProposal(proposal, 'test');
-    // 'rm' to remove the container when it exits
-    const cmd = `docker run --rm ${name}`;
-    execSync(cmd, { stdio: 'inherit' });
-  }
+export const runTestImage = (proposal: ProposalInfo) => {
```

Comment:
the API change from `runTestImages` to `runTestImage` sets off my backward-compatibility sensor.
But I don't suppose that's all that relevant in this context.

Reply:
the CLI's modules aren't part of the package API.

The package doesn't yet define `"exports"` so they technically could be imported somewhere, but I think it's a reasonable expectation that `cli` not be.

That said, this package doesn't claim backwards compatibility because, following semver, it's not yet 1.0.